PDFUnblocker.py

```
@@ -0,0 +1,63 @@
+from pathlib import Path
+import time
+import sys
+import concurrent.futures
+import pikepdf
+from tkinter import Tk
+from tkinter.filedialog import askdirectory
+from tqdm import tqdm
+
+# DEFINE FUNCTIONS HERE
+
+def SingleUnlock(filename):
```

Ruff

Missing return type annotation for public function `SingleUnlock`

Add return type annotation: `None`

(ANN201)